docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,254 @@
+<div id="toggle-button">
+    {% include section-header.html name="toggle-button" version=page.versions.toggle-button %}
+
+    <p>The Toggle Button is a button that can be toggled/pressed on or off. It is an atomic piece of the Toggle Button Group. Titles are required and other elements, such as subtitles and images can be included as well.</p>
+
+    <p>For detailed requirements and usage details, please review the <a href="https://ebay.gitbook.io/mindpatterns/input/toggle-button">eBay MIND Pattern for Toggle Button</a></p>
+
+    <p>To use these buttons as a set of multi-selection or single-selection variations, use a <span class="highlight">&lt;ul aria-label="Single-selection of payment options"&gt;</span> with buttons inside <span class="highlight">&lt;li&gt;</span>.</p>
+
+    <p><strong>NOTE:</strong> If you're using the Skin library in standalone capacity, you'll have to manually set the button toggled/pressed state using <span class="highlight">JavaScript</span> on your own by applying/changing <span class="highlight">aria-pressed="true"</span> when toggled on and <span class="highlight">aria-pressed="false"</span> when off.</p>
+
+    <h3>Layout Themes</h3>
+
+    <p>The default layout is the list view theme. To use a different layout theme, use the modifier on a button. See each layout theme for each class.</p>
+
+    <h4>Minimal View Toggle Button</h4>
+
+    <div class="demo">
+        <div class="demo__inner">
```

Comment:
To be consistent with rest of docs, we typically start with the example of the class that has no modifier (so list view in this case). Then we introduce each new modifier in logical order.